cellmlmanip/parser.py

```diff
@@ -101,7 +104,7 @@ def __init__(self, filepath):
         # A dictionary mapping component names to _Component objects
         self.components = {}
 
-    def parse(self, unit_store=None):
+    def parse(self, unit_store=None, skip_singularity_fixes=False):
```

Comment:
Can we make `fix_singularities` a method of `Model` that the user calls explicitly? I don't think any of the new code below has anything to do with parsing, so doesn't belong in the parser

Reply:
That would give you a place to document the fixes as well, and explain that you need to oxmeta annotations for this to work etc.